package.json

```diff
@@ -209,10 +208,10 @@
     "webpack-http-server": "^0.5.0"
   },
   "peerDependencies": {
-    "typescript": ">= 4.8.x"
```

Comment:
We still need to keep TypeScript. I don't see how your proposal implies it has to be removed.